opencxl/apps/multi_logical_device.py

```diff
@@ -25,12 +26,15 @@ def __init__(
         host: str = "0.0.0.0",
         port: int = 8000,
         test_mode: bool = False,
-        cxl_connections=None,
+        cxl_connections: List[CxlConnection] = None,
     ):
         label = f"Port{port_index}"
         super().__init__(label)
 
-        self._cxl_type3_devices = []
+        self._cxl_type3_devices: list[CxlType3Device] = []
+        # TODO: we also need a FMLD connection for cases like
+        # TunnelManagementCommand MCTP requests/responses
+        # See Figure 7-20/7-23 for details
```

Comment:
No need for a new connection, but would need to add a special handling for `ld-id` 0xFFFF. Please remove the comment to prevent confusion.